HW-11/Task_2.js

```diff
@@ -0,0 +1,140 @@
+/*1. Реализуйте метод findEmployeeByName(firstName: string) в классе Company, который возвращает объект сотрудника, если такой найден. */
+
+class Employee {
+    #salary;
+     constructor(firstName,lastName, profession, salary) {
+this.firstName = firstName;
+ this.lastName = lastName;
+ this.profession = profession;
+this.#salary = salary;
+     }
+get firstName() {
+    return this._firstName;
+  }
+  set firstName(firstName) {
+       if ((typeof firstName !== "string") || (!(/^[A-Za-z]+$/).test(firstName)) || (firstName.length < 2 || firstName.length > 50)) {
```

Comment:
Кажется чуть лучше вьіглядит
Как вариант можно в приватньій метод вьінести проверку и обращаться к нему
```suggestion
       if (!(/^[a-zA-Z]{2,50}$/).test(firstName)) {
```